src/components/CartCard.js

```diff
@@ -44,17 +44,17 @@ function CartCard({_id,src,title,categoryName,originalPrice,price,discount,count
                     </div>
                     <div className="quantity-text flex align-center">
                         <p>Quantity :</p>
-                        <button className="quantity-btn flex justify-center align-center" onClick={()=>cartDispatch({type:"INCREMENT_QUANTITY",payload:{_id:_id,price:Number(price),discount:Number(originalPrice)}})}>+</button>
+                        <button className="quantity-btn flex justify-center align-center" onClick={()=>cartDispatch({type:"INCREMENT_QUANTITY",payload:{_id:_id}})}>+</button>
                         <small>{count}</small>
```

Comment:
why _id as object giving directly can give

Reply:
actually i was sending few more things in payload too.